libs/application/templates/general-petition/src/fields/EndorsementApproved/index.tsx

```diff
@@ -0,0 +1,4135 @@
+import React from 'react'
+import { Box, Button, Link } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { Approved } from '@island.is/application/ui-components'
+import { m } from '../../lib/messages'
+
+const EndorsementApproved = () => {
+  const { formatMessage } = useLocale()
+
+  return (
+    <Box>
+      <Box marginTop={5} marginBottom={12}>
+        <Approved
+          title={formatMessage(m.endorsementApproved.cardTitle)}
+          subtitle={formatMessage(m.endorsementApproved.cardSubtitle)}
+        />
+      </Box>
+      <Box
+        display="flex"
+        flexDirection="column"
+        justifyContent="flexEnd"
+        alignItems="flexEnd"
+        height="full"
+        marginTop={8}
+        marginBottom={10}
+      >
+        <svg
```

Comment:
Can this SVG be made an asset and not stored in code?

Reply:
100 %